### ISSUE TITLE
2015 Sharp TVs appear to have broken IP CONTROL

### ISSUE THREAD

**Issue by golliher** [benburkhart1/sharp-aquos-remote-control#2]
Hi, 

I have written a similar tool to teach myself golang and to have fun with my TV.   I had so much fun I bought a second newer TV.    I've since discovered a problem.   At least my 2015 TV has problems and won't work with your library (or my software either).  

I have discovered that my 2015 set offers a limited version of IP CONTROL and I'm documenting what I find as an the issue in my project:  https://github.com/golliher/go-sharptv/issues/1

I'm sharing this in the hopes that it is useful to you and that if one of us finds a solution we can share information.

**Comment by golliher** [comment]
So it's as simple as this:   Not all Sharp smart TVs have implemented the IP CONTROL protocol.   There's nothing to fix in my project or yours.  Might as well close this issue.

**Comment by benburkhart1** [comment]
Hey Darrell, I'd love to play with it, I just do not have the hardware,
would we be able to setup a SSH tunnel or something so that I may access it
remotely?

**Comment by golliher** [comment]
Hi Ben,

After goign back and forth with Sharp I learned that the TV simple wasn’t compatible with IP control and that they had no intention of making it compatible with future firmware updates.    That being the case, I returned the TV and consider it case closed.

If I still had one myself, I’d totally open up a tunnel for you.     :-)

My comment was meant to put a bow on the issue I’d raised of incompatibility since I don’t see anything we can do to make software work with TVs that Sharp isn’t supporting.   It appears they are reserving IP control to only 70” and larger sets.  

Cheers,

-Darrell

—http://golliher.net
